frontend/javascripts/libs/react_helpers.tsx

```diff
@@ -67,4 +67,22 @@ export function usePolledState(callback: (arg0: OxalisState) => void, interval:
     callback(state);
   }, interval);
 }
+
+export function makeModalLazy<T extends { isVisible: boolean }>(
```

Comment:
There's no reason this wouldn't work for any React element, right? Maybe rename to `makeComponentLazy`?

Reply:
The component needs to have a `isVisible` prop, but you are right. It's not special to modals. I'll rename it :)